lib/ctf-writer/stream-class.c

```diff
@@ -232,7 +232,7 @@ int try_map_clock_class(struct bt_ctf_stream_class *stream_class,
 	}
 
 	BT_ASSERT(((struct bt_field_type_common *) ft)->id ==
-		BT_CTF_FIELD_TYPE_ID_INTEGER);
+		BT_FIELD_TYPE_ID_INTEGER);
```

Comment:
Yes, or `(int) BT_CTF_FIELD_TYPE_ID_INTEGER` would also work.

Reply:
Changed to the BT_CTF_FIELD_TYPE_ID_INTEGER option.